src/main/java/reposense/system/CommandRunner.java

```diff
@@ -31,7 +30,13 @@ public static void checkOut(String root, String hash) {
 
     public static String blameRaw(String root, String fileDirectory) {
         File rootFile = new File(root);
-        return runCommand(rootFile, "git blame " + addQuote(fileDirectory) + " -w -C -C -M --line-porcelain");
+        String blameCommand = "git blame " + addQuote(fileDirectory) + " -w -C -C -M --line-porcelain | ";
+        if (isWindows) {
+            blameCommand += "findstr /B /C:" + addQuote("author ");
+        } else {
+            blameCommand += "grep " + addQuote("^author .*");
```

Comment:
grep isn't installed by default for some linux system though. So if we use grep, we need to include it in the user guide & developer guide as well.